gbpservice/nfp/configurator/api/v1/controllers/controller.py

```diff
@@ -103,6 +102,8 @@ def post(self, **body):
             pecan.response.status = 400
             msg = ("Failed to serve HTTP post request %s %s."
                    % (self.method_name, str(err).capitalize()))
+            extra_import = ("need to remove this import %s" % config)
```

Comment:
Did you check how this is working for config agent? Closing this PR is dependent on this.